tests/build-arg-explicit-global.earth

```diff
@@ -0,0 +1,35 @@
+VERSION --explicit-global 0.6
+FROM alpine:3.13
+WORKDIR /test
+
+ARG --global global1=abc
+ARG --global global2=def
+ARG local=ghi
+
+test:
+    BUILD +test-global1
+    BUILD +test-global2
+    BUILD +test-global3
+
+dummy:
+    FROM alpine:3.13
+
+test-global1:
+    RUN test "$global1" == "abc"
+    RUN test "$global2" == "def"
+    RUN test "$local"   == ""
+
+test-global2:
+    FROM alpine:latest
+    RUN test "$global1" == "abc"
+    RUN test "$global2" == "def"
+    RUN test "$local"   == ""
+
+test-global3:
+    FROM +dummy
+    RUN test "$global1" == "abc"
+    RUN test "$global2" == "def"
+    RUN test "$local"   == ""
+
+test-failure:
+    ARG --global global1=123
```

Comment:
nit: our linter will fail on this not ending with a newline.

Reply:
thanks, I had a problem running all the tests locally so I waited for the CI to let me know how I was doing with the tests.